package.json

```diff
@@ -47,5 +47,8 @@
 		"sinon": "^18.0.0",
 		"sinon-chai": "^4.0.0",
 		"uvu": "^0.5.6"
+	},
+	"overrides": {
+		"@web/dev-server-core": "0.7.1"
```

Comment:
Bug with wtr -- failed sinon expectations aren't picked up as test failures (outside of manual mode, anyways), resulting in "locked-up" suites that rely on timing out. Downgrading for now helps.

See https://github.com/modernweb-dev/web/issues/2772